src/python/test/test_representations_interface.py

```diff
@@ -0,0 +1,90 @@
+from copy import deepcopy
```

Comment:
https://scikit-learn.org/stable/modules/generated/sklearn.base.clone.html ?
(deepcopy should be fine, but if sklearn recommends that one...)